commit 09e5e3e403b74e3bc86365fe356fd9fb9690bd24
Author: Jacob Abel <thatcadguy@gmail.com>
Date:   Sat May 3 07:16:47 2014 -0700

    [clang-warning-fix][logical-not-parentheses]
    
    Since gce_Done is an enumerated type, what's there makes no sense at all
    and was surely meant to be what I've changed it to.

Index: oce/src/GC/GC_MakeArcOfCircle.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeArcOfCircle.cxx
+++ oce/src/GC/GC_MakeArcOfCircle.cxx
@@ -158,7 +158,7 @@ GC_MakeArcOfCircle::GC_MakeArcOfCircle(c
 //=======================================================================
 const Handle(Geom_TrimmedCurve)& GC_MakeArcOfCircle::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheArc;
 }
 //=======================================================================
Index: oce/src/GC/GC_MakeArcOfEllipse.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeArcOfEllipse.cxx
+++ oce/src/GC/GC_MakeArcOfEllipse.cxx
@@ -54,7 +54,7 @@ GC_MakeArcOfEllipse::GC_MakeArcOfEllipse
 
 const Handle(Geom_TrimmedCurve)& GC_MakeArcOfEllipse::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheArc;
 }
 
Index: oce/src/GC/GC_MakeArcOfHyperbola.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeArcOfHyperbola.cxx
+++ oce/src/GC/GC_MakeArcOfHyperbola.cxx
@@ -55,7 +55,7 @@ GC_MakeArcOfHyperbola::GC_MakeArcOfHyper
 
 const Handle(Geom_TrimmedCurve)& GC_MakeArcOfHyperbola::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheArc;
 }
 
Index: oce/src/GC/GC_MakeArcOfParabola.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeArcOfParabola.cxx
+++ oce/src/GC/GC_MakeArcOfParabola.cxx
@@ -54,7 +54,7 @@ GC_MakeArcOfParabola::GC_MakeArcOfParabo
 
 const Handle(Geom_TrimmedCurve)& GC_MakeArcOfParabola::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheArc;
 }
 
Index: oce/src/GC/GC_MakeCircle.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeCircle.cxx
+++ oce/src/GC/GC_MakeCircle.cxx
@@ -97,7 +97,7 @@ GC_MakeCircle::GC_MakeCircle(const gp_Ax
 
 const Handle(Geom_Circle)& GC_MakeCircle::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCircle;
 }
 
Index: oce/src/GC/GC_MakeConicalSurface.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeConicalSurface.cxx
+++ oce/src/GC/GC_MakeConicalSurface.cxx
@@ -105,7 +105,7 @@ GC_MakeConicalSurface::GC_MakeConicalSur
 
 const Handle(Geom_ConicalSurface)& GC_MakeConicalSurface::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCone;
 }
 
Index: oce/src/GC/GC_MakeCylindricalSurface.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeCylindricalSurface.cxx
+++ oce/src/GC/GC_MakeCylindricalSurface.cxx
@@ -98,7 +98,7 @@ GC_MakeCylindricalSurface::GC_MakeCylind
 const Handle(Geom_CylindricalSurface)& 
        GC_MakeCylindricalSurface::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCylinder;
 }
 
Index: oce/src/GC/GC_MakeEllipse.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeEllipse.cxx
+++ oce/src/GC/GC_MakeEllipse.cxx
@@ -48,7 +48,7 @@ GC_MakeEllipse::GC_MakeEllipse(const gp_
 
 const Handle(Geom_Ellipse)& GC_MakeEllipse::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheEllipse;
 }
 
Index: oce/src/GC/GC_MakeHyperbola.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeHyperbola.cxx
+++ oce/src/GC/GC_MakeHyperbola.cxx
@@ -47,7 +47,7 @@ GC_MakeHyperbola::GC_MakeHyperbola(const
 
 const Handle(Geom_Hyperbola)& GC_MakeHyperbola::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheHyperbola;
 }
 
Index: oce/src/GC/GC_MakeLine.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeLine.cxx
+++ oce/src/GC/GC_MakeLine.cxx
@@ -63,7 +63,7 @@ GC_MakeLine::GC_MakeLine(const gp_Lin& L
 
 const Handle(Geom_Line)& GC_MakeLine::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheLine;
 }
 
Index: oce/src/GC/GC_MakePlane.cxx
===================================================================
--- oce.orig/src/GC/GC_MakePlane.cxx
+++ oce/src/GC/GC_MakePlane.cxx
@@ -119,7 +119,7 @@ GC_MakePlane::GC_MakePlane(const gp_Ax1&
 
 const Handle(Geom_Plane)& GC_MakePlane::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return ThePlane;
 }
 
Index: oce/src/GC/GC_MakeSegment.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeSegment.cxx
+++ oce/src/GC/GC_MakeSegment.cxx
@@ -61,7 +61,7 @@ GC_MakeSegment::GC_MakeSegment(const gp_
 
 const Handle(Geom_TrimmedCurve)& GC_MakeSegment::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheSegment;
 }
 
Index: oce/src/GC/GC_MakeTrimmedCone.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeTrimmedCone.cxx
+++ oce/src/GC/GC_MakeTrimmedCone.cxx
@@ -73,7 +73,7 @@ GC_MakeTrimmedCone::GC_MakeTrimmedCone(c
 const Handle(Geom_RectangularTrimmedSurface)& GC_MakeTrimmedCone::
        Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCone;
 }
 
Index: oce/src/GC/GC_MakeTrimmedCylinder.cxx
===================================================================
--- oce.orig/src/GC/GC_MakeTrimmedCylinder.cxx
+++ oce/src/GC/GC_MakeTrimmedCylinder.cxx
@@ -82,7 +82,7 @@ GC_MakeTrimmedCylinder::GC_MakeTrimmedCy
 const Handle(Geom_RectangularTrimmedSurface)& GC_MakeTrimmedCylinder::
        Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCyl;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeArcOfCircle.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeArcOfCircle.cxx
+++ oce/src/GCE2d/GCE2d_MakeArcOfCircle.cxx
@@ -111,7 +111,7 @@ GCE2d_MakeArcOfCircle::GCE2d_MakeArcOfCi
 
 const Handle(Geom2d_TrimmedCurve)& GCE2d_MakeArcOfCircle::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheArc;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeArcOfEllipse.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeArcOfEllipse.cxx
+++ oce/src/GCE2d/GCE2d_MakeArcOfEllipse.cxx
@@ -54,7 +54,7 @@ GCE2d_MakeArcOfEllipse::GCE2d_MakeArcOfE
 
 const Handle(Geom2d_TrimmedCurve)& GCE2d_MakeArcOfEllipse::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheArc;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeArcOfHyperbola.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeArcOfHyperbola.cxx
+++ oce/src/GCE2d/GCE2d_MakeArcOfHyperbola.cxx
@@ -56,7 +56,7 @@ GCE2d_MakeArcOfHyperbola::
 
 const Handle(Geom2d_TrimmedCurve)& GCE2d_MakeArcOfHyperbola::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheArc;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeArcOfParabola.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeArcOfParabola.cxx
+++ oce/src/GCE2d/GCE2d_MakeArcOfParabola.cxx
@@ -55,7 +55,7 @@ GCE2d_MakeArcOfParabola::
 
 const Handle(Geom2d_TrimmedCurve)& GCE2d_MakeArcOfParabola::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheArc;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeCircle.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeCircle.cxx
+++ oce/src/GCE2d/GCE2d_MakeCircle.cxx
@@ -100,7 +100,7 @@ GCE2d_MakeCircle::GCE2d_MakeCircle(const
 
 const Handle(Geom2d_Circle)& GCE2d_MakeCircle::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCircle;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeEllipse.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeEllipse.cxx
+++ oce/src/GCE2d/GCE2d_MakeEllipse.cxx
@@ -59,7 +59,7 @@ GCE2d_MakeEllipse::GCE2d_MakeEllipse(con
 
 const Handle(Geom2d_Ellipse)& GCE2d_MakeEllipse::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheEllipse;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeHyperbola.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeHyperbola.cxx
+++ oce/src/GCE2d/GCE2d_MakeHyperbola.cxx
@@ -59,7 +59,7 @@ GCE2d_MakeHyperbola::GCE2d_MakeHyperbola
 
 const Handle(Geom2d_Hyperbola)& GCE2d_MakeHyperbola::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheHyperbola;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeLine.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeLine.cxx
+++ oce/src/GCE2d/GCE2d_MakeLine.cxx
@@ -73,7 +73,7 @@ GCE2d_MakeLine::GCE2d_MakeLine(const gp_
 
 const Handle(Geom2d_Line)& GCE2d_MakeLine::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheLine;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeParabola.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeParabola.cxx
+++ oce/src/GCE2d/GCE2d_MakeParabola.cxx
@@ -74,7 +74,7 @@ GCE2d_MakeParabola::GCE2d_MakeParabola(c
 
 const Handle(Geom2d_Parabola)& GCE2d_MakeParabola::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheParabola;
 }
 
Index: oce/src/GCE2d/GCE2d_MakeSegment.cxx
===================================================================
--- oce.orig/src/GCE2d/GCE2d_MakeSegment.cxx
+++ oce/src/GCE2d/GCE2d_MakeSegment.cxx
@@ -77,7 +77,7 @@ GCE2d_MakeSegment::GCE2d_MakeSegment(con
 
 const Handle(Geom2d_TrimmedCurve)& GCE2d_MakeSegment::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheSegment;
 }
 
Index: oce/src/gce/gce_MakeCirc.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeCirc.cxx
+++ oce/src/gce/gce_MakeCirc.cxx
@@ -338,7 +338,7 @@ gce_MakeCirc::gce_MakeCirc(const gp_Circ
 //=======================================================================
 const gp_Circ& gce_MakeCirc::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCirc;
 }
 //=======================================================================
Index: oce/src/gce/gce_MakeCirc2d.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeCirc2d.cxx
+++ oce/src/gce/gce_MakeCirc2d.cxx
@@ -202,7 +202,7 @@ gce_MakeCirc2d::gce_MakeCirc2d(const gp_
 
 const gp_Circ2d& gce_MakeCirc2d::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCirc2d;
 }
 
Index: oce/src/gce/gce_MakeCone.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeCone.cxx
+++ oce/src/gce/gce_MakeCone.cxx
@@ -198,7 +198,7 @@ gce_MakeCone::gce_MakeCone(const gp_Pnt&
 
 const gp_Cone& gce_MakeCone::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCone;
 }
 
Index: oce/src/gce/gce_MakeCylinder.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeCylinder.cxx
+++ oce/src/gce/gce_MakeCylinder.cxx
@@ -126,7 +126,7 @@ gce_MakeCylinder::gce_MakeCylinder(const
 
 const gp_Cylinder& gce_MakeCylinder::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheCylinder;
 }
 
Index: oce/src/gce/gce_MakeDir.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeDir.cxx
+++ oce/src/gce/gce_MakeDir.cxx
@@ -63,7 +63,7 @@ gce_MakeDir::gce_MakeDir(const Standard_
 
 const gp_Dir& gce_MakeDir::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheDir;
 }
 
Index: oce/src/gce/gce_MakeDir2d.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeDir2d.cxx
+++ oce/src/gce/gce_MakeDir2d.cxx
@@ -62,7 +62,7 @@ gce_MakeDir2d::gce_MakeDir2d(const Stand
 
 const gp_Dir2d& gce_MakeDir2d::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheDir2d;
 }
 
Index: oce/src/gce/gce_MakeElips.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeElips.cxx
+++ oce/src/gce/gce_MakeElips.cxx
@@ -63,7 +63,7 @@ gce_MakeElips::gce_MakeElips(const gp_Pn
 
 const gp_Elips& gce_MakeElips::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheElips;
 }
 
Index: oce/src/gce/gce_MakeElips2d.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeElips2d.cxx
+++ oce/src/gce/gce_MakeElips2d.cxx
@@ -69,7 +69,7 @@ gce_MakeElips2d::gce_MakeElips2d(const g
 
 const gp_Elips2d& gce_MakeElips2d::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheElips2d;
 }
 
Index: oce/src/gce/gce_MakeHypr.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeHypr.cxx
+++ oce/src/gce/gce_MakeHypr.cxx
@@ -55,7 +55,7 @@ gce_MakeHypr::gce_MakeHypr(const gp_Ax2&
 
 const gp_Hypr& gce_MakeHypr::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheHypr;
 }
 
Index: oce/src/gce/gce_MakeHypr2d.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeHypr2d.cxx
+++ oce/src/gce/gce_MakeHypr2d.cxx
@@ -67,7 +67,7 @@ gce_MakeHypr2d::gce_MakeHypr2d(const gp_
 
 const gp_Hypr2d& gce_MakeHypr2d::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheHypr2d;
 }
 
Index: oce/src/gce/gce_MakeLin.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeLin.cxx
+++ oce/src/gce/gce_MakeLin.cxx
@@ -69,7 +69,7 @@ gce_MakeLin::gce_MakeLin(const gp_Lin& L
 
 const gp_Lin& gce_MakeLin::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheLin;
 }
 
Index: oce/src/gce/gce_MakeLin2d.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeLin2d.cxx
+++ oce/src/gce/gce_MakeLin2d.cxx
@@ -103,7 +103,7 @@ gce_MakeLin2d::gce_MakeLin2d(const gp_Li
 
 gp_Lin2d gce_MakeLin2d::Value() const
 { 
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheLin2d;
 }
 
Index: oce/src/gce/gce_MakeParab.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeParab.cxx
+++ oce/src/gce/gce_MakeParab.cxx
@@ -37,7 +37,7 @@ gce_MakeParab::gce_MakeParab(const gp_Ax
 
 const gp_Parab& gce_MakeParab::Value () const
 {
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheParab;
 }
 
Index: oce/src/gce/gce_MakeParab2d.cxx
===================================================================
--- oce.orig/src/gce/gce_MakeParab2d.cxx
+++ oce/src/gce/gce_MakeParab2d.cxx
@@ -75,7 +75,7 @@ gce_MakeParab2d::gce_MakeParab2d(const g
 
 const gp_Parab2d& gce_MakeParab2d::Value () const
 {
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return TheParab2d;
 }
 
Index: oce/src/gce/gce_MakePln.cxx
===================================================================
--- oce.orig/src/gce/gce_MakePln.cxx
+++ oce/src/gce/gce_MakePln.cxx
@@ -124,7 +124,7 @@ gce_MakePln::gce_MakePln(const gp_Ax1& A
 */
 const gp_Pln& gce_MakePln::Value () const
 {
-  StdFail_NotDone_Raise_if(!TheError == gce_Done,"");
+  StdFail_NotDone_Raise_if(!(TheError == gce_Done),"");
   return ThePln;
 }
 
